apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -198,6 +320,11 @@ private static string LogSimpleLogEntry(GoogleCloudConsoleFormatterOptions optio
     {
         // Create a simple log entry and return its Json string based on specified parameters.
         LogEntry<string> logEntry = new LogEntry<string>(LogLevel.Information, "LogCategory", new EventId(1), "test", exception: null, (state, exception) => state);
+        return LogLogEntry(logEntry, options, scopeProvider);
+    }
+
+    private static string LogLogEntry<T>(LogEntry<T> logEntry, GoogleCloudConsoleFormatterOptions options, IExternalScopeProvider scopeProvider = null)
+    {
```

Comment:
I think you didn't needed this refactor in the end, you are just using LogSimpleLogEntry as far as I can see?

Reply:
true :)

Reply:
true :)